system-tests/extends_airbnb/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   extends: ["airbnb"],
-  parser: "babel-eslint",
+  parser: "@babel/eslint-parser",
+  parserOptions: {
+    requireConfigFile: false,
+  },
```

Comment:
same as above, it's not clear to me why is this needed at least not after reading the eslint 8 migration document.